stable/mysql/templates/secrets.yaml

```diff
@@ -11,16 +12,21 @@ metadata:
     heritage: "{{ .Release.Service }}"
 type: Opaque
 data:
+  {{ if .Values.useMysqlRootPassword }}
   {{ if .Values.mysqlRootPassword }}
```

Comment:
I think perhaps it is more intuitive to add `or .Values.allowEmptyRootPassword` here to prevent the random string from being generated.

Reply:
Good catch. Fixed!